### ISSUE TITLE
Proposal: skip certain functions for no-magic-numbers

### ISSUE THREAD

**Issue by KidsKilla** [eslint/eslint#10723]
**Eslint version**
5.6.0

**What rule do you want to change?**
no-magic-numbers

**Does this change cause the rule to produce more or fewer warnings?**
fewer false-positives

**How will the change be implemented? (New option, new default behavior, etc.)?**
new option: `ignoreFunctions: ['normalizeValue']`

**Please provide some example code that this change will affect:**

```js
const Comp = styled({
  marginTop: normalizeValue(3),
})
```

**What does the rule currently do for this code?**
allows to skip certain functions which accepts numeric values: css, configs

**What will the rule do after it's changed?**
allows to enable this rule but skip false-positive warnings for certain functions

We have a function that comes with our design system. It accepts ints and converts it to actual values. So it is used for styling, hence used quite extensively. But in all other cases we don't want to set values directly.
